artifactory.py

```diff
@@ -1313,6 +1313,23 @@ def parent(self):
         obj.timeout = self.timeout
         return obj
 
+    @property
+    def replication_status(self):
+        """
+        Get status of the repo replication
+        :return:
+            namedtuple('status' (bool): True only if OK, 'details' (dict): full response
+        """
+        replication_url = self.drive + "/api/replication/" + self.repo
+        replication_obj = self.joinpath(replication_url)
+        resp = self._accessor.get_response(replication_obj).json()
+
+        success = True if resp["status"] == "ok" else False
```

Comment:
`success = resp["status"] == "ok` give the same result

Reply:
fixed